### DISCUSSION TITLE
load_filament hotend position

### DISCUSSION THREAD

**Discussion by jonneybarvo** [jschuh/klipper-macros#136] [Q&A] [answered]
Hi jschuh, can the load_filament position be changed in the start_print?  I would like the hotend to move front and centre for easy filament loading.......

**Comment by jschuh** [comment]
Right now it just uses the configured parking position. I'd rather not add another config value just for a load/unload parking position, but I could update the load/unload macros to accept XY parking coordinates, which you could just override to your preferred default.

**Comment by jonneybarvo** [comment]
That sounds great 👍﻿

**Comment by jonneybarvo** [comment]
by the way - using your macros - working out great - thanks﻿

**Comment by jschuh** [comment, marked as answer]
I forgot that LOAD_FILAMENT and UNLOAD_FILAMENT don't actually change the extruder position, and I really don't think they should. You can just wrap it with something like this if you want:

```
[gcode_macro LOAD_FILAMENT]
rename_existing: LOAD_FILAMENT_BASE
gcode:
  PARK LAZY=1 X=<add your X or remove for default> Y=<add your Y or remove for default>
  LOAD_FILAMENT_BASE {rawparams}
```

**Comment by jonneybarvo** [comment]
Thanks for the advice - will give that a go - I am only starting to get my
head around macros.  Cheers﻿
